fe/fe-core/src/main/java/com/starrocks/catalog/ScalarType.java

```diff
@@ -50,9 +50,9 @@ public class ScalarType extends Type implements Cloneable {
     // precision that is supported by the smallest decimal type in the BE (4 bytes).
     public static final int DEFAULT_PRECISION = 9;
     public static final int DEFAULT_SCALE = 0; // SQL standard
-    // Longest supported VARCHAR and CHAR, chosen to match Hive.
-    public static final int DEFAULT_STRING_LENGTH = 65533;
     public static final int MAX_VARCHAR_LENGTH = 1048576;
+    // Longest supported VARCHAR and CHAR, chosen to match Hive.
+    public static final int DEFAULT_STRING_LENGTH = MAX_VARCHAR_LENGTH;
```

Comment:
Does it affect native table?

Reply:
Native table don't have string type, this variable only used for the external table.